src/pathfora.js

```diff
@@ -1437,24 +1412,25 @@
 
           // The top recommendation should be default if we couldn't
           // get one from the api
-          var rec = config.content[0]
+          var rec = config.content[0],
+              recImage = document.createElement('div'),
+              recMeta = document.createElement('div'),
+              recTitle = document.createElement('h4'),
+              recDesc = document.createElement('p');
+
           widgetContentUnit.href = rec.url;
 
-          var recImage = document.createElement('div');
```

Comment:
Added a few, though for the most part a lot of the vars were already defined at the top even before the linting. Let me know if/where you see places that may need more context.